### PR TITLE
chore: adding pretty-quick

### PR DESCRIPTION
**Pull request by jrenee42**
very simple.

not bumping the version number since I'll be doing another pr  for annotations and will bump it then.

### CODE REVIEW

File: giraffe/package.json [hunk 1]
```diff
@@ -107,6 +107,7 @@
     "react-dom": "^16.8.0"
   },
   "dependencies": {
-    "merge-images": "^2.0.0"
+    "merge-images": "^2.0.0",
+    "pretty-quick": "^3.1.0"
```
Comment (TCL735) [inline giraffe/package.json:111]: This should be in `devDependencies` just like `prettier` is.

**Review by TCL735 — CHANGES_REQUESTED** [review]

File: giraffe/package.json (hunk 1 above)
Comment (jrenee42) [reply giraffe/package.json]: got it.  manually moved it over.

File: giraffe/package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "build": "rm -rf dist && NODE_ENV=production webpack --config webpack.config.js",
     "lint": "eslint '{src,../stories/src}/**/*.{ts,tsx}'",
-    "prettier": "prettier --config ../.prettierrc.json --check '{src,../stories/src}/**/*.{ts,tsx}'",
-    "prettier:fix": "prettier --config ../.prettierrc.json --write '{src,../stories/src}/**/*.{ts,tsx}'",
+    "prettier": "pretty-quick --config ../.prettierrc.json --check '{src,../stories/src}/**/*.{ts,tsx}'",
+    "prettier:fix": "pretty-quick --config ../.prettierrc.json --write '{src,../stories/src}/**/*.{ts,tsx}'",
```
Comment (TCL735) [inline giraffe/package.json:11]: I would like to preserve the original scripts as well. The originals can have new names. Something like `prettier:all` and `prettier:all:fix`

Comment (TCL735) [reply giraffe/package.json:11]: This would be the last issue and then this PR will be good to go. Thanks.

**Review by TCL735 — CHANGES_REQUESTED** [review]

**Review by TCL735 — APPROVED** [review]

File: giraffe/package.json (hunk 2 above)
Comment (jrenee42) [reply giraffe/package.json:11]: ps:  none of the other projects preserve the original scripts.  so you can add them back if you like. (clockface and ui)

**Pull request merged**